controllers/configurationpolicy_controller.go

```diff
@@ -983,13 +995,21 @@ func (r *ConfigurationPolicyReconciler) handleObjects(
 
 		log.V(2).Info("Handling a single object template")
 
-		objNames, compliant, rsrcKind, statusUpdateNeeded = r.handleSingleObj(
+		_, compliant, _, _, creationInfo := r.handleSingleObj(
```

Comment:
I'm not following why some of these are now `_`... previously some of these values were being updated by what was coming from `handleSingleObj`. If they no longer need to be updated, I think this is the only place that function is called, so we could just remove them from the function signature.

Or maybe this is appearing because of variable shadowing? Were there unused variable errors before changing them to `_`? In which case, I think that `creationInfo` should be initialized separately, then this can be remain a `=` statement, instead of `:=`.

Reply:
I thought these variables weren't actually used in that block but I might have been wrong. I will look into it